### ISSUE TITLE
Improve Knightmetal Sources

### ISSUE THREAD

**Issue by Sindusk** [GTNewHorizons/GT-New-Horizons-Modpack#12222]
### Your GTNH Discord Username

Sindusk#7799

### Your Pack Version

2.2.0.0

### Your Proposal

Knightmetal (armor shard) can only be obtained in reasonable quantity by finding a spawner for a Helmet Crab. These occur in the dungeons beneath the Dark Forest, but are not guaranteed. This means that a player can clear several dungeons and find no Helmet Crabs, leading to a significant lack of knightmetal if they don't generate. In turn, this hinders thaumcraft progression, which requires Knightmetal screws to create thaumium caps. This can also inhibit the player's ability to grow knightmetal oreberry after breeding them, as it requires a full block of Knightmetal in order to grow them to full maturity.

As it stands, the two previous dungeons follow a pattern of maceration. The gear found there can be macerated for the material required. In the Lich's Tower, you can find Ironwood gear. This can be macerated to Ironwood Dust and then smelted to Ironwood Ingots. In the Labyrinth, you are able to find Steeleaf gear. This is able to be macerated to Steeleaf Dust then alloy smelted in ingot form to a Steeleaf.

In contrast, the Knightmetal dungeon does not provide the same courtesy. Instead, the armors that are found as loot are "Phantom" instead of "Knightly." This phantom gear is unable to be macerated for anything, and as far as I can tell have no recipes whatsoever. Additionally, the "Knightly" sword, pick, and axe have no recipes either, further exacerbating a void of materials that should be provided from clearing the dungeon.

As a final black mark, the loot from the Lich and Labyrinth found in chests are full ingots of Ironwood and Steeleaf. In stark contrast, the loot in the dark forest dungeon is "Armor Shard." This has a 9:1 recipe to "Armor Shard Cluster" which can then be smelted into a single Knightmetal ingot. This makes Knightmetal 9x as rare as the two previous resources if the loot tables for chests are identical.

**Proposal:** Add a recipe to macerate the Phantom armor and Knightly sword, pick, and axe into Knightmetal Dust.

### Your Goal

This change would eliminate bad RNG for dungeon layouts creating an artificial hinderance to Thaumcraft progression.

### Your Vision

Thaumium Caps are not required in huge amounts (unless for specific purposes). This change allows players who are simply attempting to progress to do so without headache. For those who require Knightmetal in significant quantities, they can go through the effort of finding a spawner and creating a powered spawner.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by LazyFleshWasTaken** [comment]
I just went ot the TF, found a maze, and first hallway chest i opened had 4 knightmetal in it. 
![image](https://user-images.githubusercontent.com/40587985/211139832-761498ae-f352-40c9-8bc8-bbe6ced70876.png)
Second of all, the knightly picks and swords you get as boss drops CAN be made into knightmetal.
![image](https://user-images.githubusercontent.com/40587985/211139896-2bcc9512-dc15-42e4-81eb-3bfa46571d59.png)
The amount you need in Thaumcraft is negligible, as it needs one ingot per 2 thaumium caps.

**Comment by Sindusk** [comment]
Worth checking the drop rates then, because I never saw a single chest with anything more than armor shards. Also I have no idea how you're able to macerate those items.
https://streamable.com/bnb54i

**Comment by chochem** [comment]
I am calling BS on this entire post.

First of all:

> Knightmetal (armor shard) can only be obtained in reasonable quantity by finding a spawner for a Helmet Crab. 

This is false and you already know that!! We literally talked about it on discord yesterday. There is a crop. And even if you dont like crops, you also dont need a spawner but just a single crab to use a soul viel.

Second: the dungeons. I just went into a fresh dungeon to see if maybe they are broken or something and guess what the first mob is that walks my way? A helmet crap. And the second? A Goblin knight. I have no idea how you managed to clear 2 full areas without seeing either. It defies believe. They also respawn all the time. You cant spend 5 minutes there without finding one.

I also found chests with knightmetal ingots/shards and two double helmet crap spawner (not that you need those). And of course the boss loot.

Haul without camping/taking the spawners: 
![image](https://user-images.githubusercontent.com/40274384/211164528-780df062-f438-43d2-84f5-9730cc2ef960.png)


Third: the video. 
This is what the bosses drop for me:
![image](https://user-images.githubusercontent.com/40274384/211164302-00674ec1-bf67-4b73-b5d3-8d7181de1af6.png)
They are all enchanted contrary to what you showed and they macerate just fine:
![image](https://user-images.githubusercontent.com/40274384/211164317-7adc965e-2a89-4ba6-a5e6-f14598f394d0.png)
are you sure your macerator is even powered?

same for the sword. 
![image](https://user-images.githubusercontent.com/40274384/211164417-37a2a38d-1298-41f3-b300-06af86b6d26f.png)
![image](https://user-images.githubusercontent.com/40274384/211164427-0b99c021-7243-4a60-8623-f32f13d648ed.png)

**Comment by Sindusk** [comment]
Our experiences are vastly different, and I'm beginning to believe this is not actually a suggestion and instead it's a bug. Here is nearly two minutes of me running through a dungeon finding absolutely nothing. There are no lights placed and I'm using nightvision:

https://streamable.com/btsrbe

Additionally, yes, of course, my macerator is powered. I took two items from the boss chest directly in that video. Here I attempt to macerate them after some stone:

https://streamable.com/iyzmu3

Your results differ from mine so dramatically it's got me questioning if we're playing the same version. Perhaps it was fixed between 2.2.0 and whatever version you're on?

**Comment by chochem** [comment]
did you enter through that sidehole? that would be your mistake then. Use the normal entrance.

**Comment by Sindusk** [comment]
I used the main entrance where you put the trophy on my first dungeon and the same thing happened. This is the second dungeon I had cleared where I went through the side entrance. In grand total I came out with 5 knightmetal ingots and 4 armor pieces in total after 2 full dungeons and killing the boss both times.

Afterwards, I went to another dungeon (the third one that I went to). It was fine, and worked exactly like you described. It had helmet crabs everywhere and full knightmetal ingots in the chests. Even saw some of the goblins for the first time. I didn't use the main entrance, either.

I'm beginning to think it's a worldgen/dungeon generation problem where some just don't function properly. But really I'm curious about why you're able to macerate the items and I cannot. Very strange.
